.travis.yml

```diff
@@ -112,18 +112,22 @@ matrix:
       script:
         - "./build/travis/job_macos/script.sh"
 
-
 notifications:
-  recipients:
-    - $NOTIFICATION_EMAIL
   email:
+    recipients:
+      - "build@musescore.org"
     on_success: change
     on_failure: always
   irc:
     channels:
-      - $NOTIFICATION_IRC
+      - "chat.freenode.net#musescore"
     on_success: change
     on_failure: change
+    template: # https://docs.travis-ci.com/user/notifications/#IRC-notification
```

Comment:
Fair enough. I would have though the message would be the most important thing since everything else is just random digits, but it's your call.